src/App/App.js

```diff
@@ -33,6 +33,7 @@ function App() {
     console.log("showMovieDetails CLICK!")
```

Comment:
This is good for working through our code, Im glad we are making use of console logs to see whats going on under the hood, well have to remember to go through and remove these before final submission